File: pyg-shop-web/src/main/webapp/js/controller/loginController.js
//定义控制器
app.controller("loginController", function ($scope,loginService) {


    //定义查询所有的方法
    $scope.showName = function () {
        //使用内置服务向后台发送ajax请求
        loginService.showLoginName().success(function (data) {
            $scope.loginName = data.loginName;
        })
    };


});

File: pyg-shop-web/src/main/java/com/pyg/shop/controller/LoginController.java
package com.pyg.shop.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/login")
public class LoginController {

    /**
     * 需求：获取用户登陆名，进行回显
     */
    @RequestMapping("showName")
    public Map showLoginName(){

        //获取用户登录名
        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        //创建map封装登陆名
        Map loginMap = new HashMap();
        loginMap.put("loginName",name);
        return loginMap;
    }
}
